### PR TITLE
updated the bot to use the new land/4 percept

### PR DESCRIPTION
**Pull request by dionytadema**
staged PR for when new land percept gets merged in the context branch

https://github.com/levilime/tygron/pull/19 is merged,

waiting for https://github.com/eishub/tygron/pull/73

### CODE REVIEW

**Comment by NickTUD** [comment]
Is it possible to add a test as well?

**Comment by dionytadema** [comment]
Yeah there was no test yet but i guess i could add one

**Comment by NickTUD** [comment]
@dionytadema poke, you need to resolve conflicts

EDIT: resolved

**Comment by NickTUD** [comment]
Why do all variable names suddenly have 2 letter names again????
And also why are you using %%% again?

File: Municipality/test.test2g [hunk 1]
```diff
@@ -10,12 +10,23 @@ test tygron with
...
+		percept(buildings(List)), bel(member(building(ID,Name,OwnerID,ConstructionYear,CategoriesList,FuncID,Floors),List)) 
+			leadsto bel(building(ID,Name,OwnerID,ConstructionYear,CategoriesList,FuncID,Floors))).
+=======
 		percept(indicators(List)),bel(member(indicator(IndicatorID,Current,Target,ZoneIndicatorsList),List)) 
 			leadsto bel(indicator(IndicatorID,Current,Target,ZoneIndicatorsList)).
 		percept(zones(List)), bel(member(zone(ZoneID,Name,Floors,Size,CategoriesList),List)) 
 			leadsto bel(zone(ZoneID,Name,Floors,Size,CategoriesList)).
 		percept(buildings(List)), bel(member(building(BuildingID,Name,OwnerID,ConstructionYear,CategoriesList,FuncID,Floors),List)) 
 			leadsto bel(building(BuildingID,Name,OwnerID,ConstructionYear,CategoriesList,FuncID,Floors))).
+>>>>>>> refs/heads/master
```
Comment (NickTUD) [inline Municipality/test.test2g:29]: Merge conflict????

**Pull request closed without merging**